.sass-lint.yml

```diff
@@ -68,7 +63,7 @@ rules:
   empty-args: 1
   hex-length: 0
   hex-notation:
-    - 2
+    - 1
```

Comment:
What is this value for? To reduce the criticality?

Reply:
yes, it used to be an error, but a warning is better in this case, because we need to solve all the warnings, and I don't want to do it in this PR